### ISSUE TITLE
accessing intercept value

### ISSUE THREAD

**Issue by saurabac** [jhelvy/logitr#23]
Hi John, this is a great package! Much faster and stable compared to the other R packages for mixed logits. Is it possible to extract the intercept for the estimated model (or a method to specify it as a parameter)?
